src/tests/fixtures/tabs.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Tabs</title>
+    <script src="/dist/turbo.es2017-umd.js"></script>
+    <script src="/src/tests/fixtures/test.js"></script>
+  </head>
+  <body>
+    <h1>Tabs</h1>
+
+    <turbo-frame id="tab-content" data-turbo-action="advance">
+      <div>
+        <a id="drive_enabled" href="/src/tests/fixtures/tabs.html">Tab 1</a>
+        <a id="drive_enabled" href="/src/tests/fixtures/tabs/two.html">Tab 2</a>
+        <a id="drive_enabled" href="/src/tests/fixtures/tabs/three.html">Tab 3</a>
```

Comment:
Is it necessary to declare these anchors with an `[id]` attribute? If it is, could we make sure their values don't collide with other elements on the page?

Also, are there plans to add tests to cover the new behavior and to guard against regressions?

Reply:
yeah, I wanted to validate the idea and implementation and will follow up with some tests

Reply:
@seanpdoyle I added some tests in e99e69e...16b3bdd, but it's really hard to know the state of tests already in place since there's a lot of flakiness. I'm different seeing test pass and fail each run